Directory.Build.Local.props

```diff
@@ -76,7 +76,7 @@
 
   <!-- Test config -->
   <PropertyGroup>
-    <TestRunnerName>RepoCustomRunner</TestRunnerName>
+    <TestRunnerName>TestingPlatformRunner</TestRunnerName>
```

Comment:
What's the reasoning here? This seems wrong. The current target was already able to run either the vstest + custom adapter for regular MSTest or TA tests for the runner. This change is breaking this.

Reply:
I was just trying to make it work in the parent repo where this name is used and was missing.

Reply:
Oh ok, I misunderstood the change you actually did. That's all good as-is, thanks!